### ISSUE TITLE
Sats for review

### ISSUE THREAD

**Issue by adamjonas** [adamjonas/transcription-queue#5]
As a reviewer, I will be paid 1 sat per word once my reviewed transcript is merged into bitcointranscripts.

This means that reviewers will be signed in via their GitHub so that we can push changes with their account but will also need to ln-auth (e.g. https://makers.bolt.fun/login) a wallet for us to stream sats to.
